apps/src/feedback.js

```diff
@@ -147,6 +147,12 @@ FeedbackUtils.prototype.displayFeedback = function(
 
   if (feedbackMessage) {
     feedback.appendChild(feedbackMessage);
+    if (options.level.skin === 'applab' && options.level.lastLevelInLesson) {
```

Comment:
Why is this specific to applab?

Reply:
Maybe a better way to ask the question is why is there applab-specific logic needed here?

Reply:
Currently, the App Lab Finish Dialog has a message explaining "With App Lab you're in control. Are you really finished...." etc. With this change that message gets overridden with the "Congratulations! You finished the lesson message" but Product still wanted the "With App Lab you're in control. Are you really finished...." messaging, just not in the purple header.

Reply:
I see, and this special case is new because previously applab could use the same message as other lab types?

It's unfortunate that we have to put applab-specific code here (I don't see applab mentioned elsewhere in this file) but I don't understand the structure of the code here to suggest a better solution.

Also, I see some logic later around `options.level.disableFinalLessonMessage`.  How does this work relate to that flag?

Reply:
Also, if there are design changes that you think we can make that would simplify the code but still meet product's goals, please suggest them!

Reply:
Got the ok to cut the AppLab-specific customization, so things should look a little simpler! 🎉 Thanks for the nudge.